client/packages/coldchain/src/Equipment/ImportAsset/EquipmentImportModal.tsx

```diff
@@ -70,12 +75,30 @@ export const toInsertEquipmentInput = (
 
   return {
     ...rest,
+    needsReplacement: !!row.needsReplacement,
     catalogueItemId,
     store: store ? { ...store, __typename: 'StoreNode', storeName: '' } : null,
     parsedProperties,
   };
 };
 
+export const toStatusTypeInput = (status: StatusType): AssetLogStatusInput => {
```

Comment:
Can we add these to the docs somewhere, so people know what options they have for that column?